RecoTauTag/Configuration/python/boostedHPSPFTaus_cfi.py

```diff
@@ -21,12 +21,9 @@ def addBoostedTaus(process):
     process.ak4PFJetsRecoTauChargedHadronsBoosted.builders[1].dRcone = cms.double(0.3)
     process.ak4PFJetsRecoTauChargedHadronsBoosted.builders[1].dRconeLimitedToJetArea = cms.bool(True)
     process.combinatoricRecoTausBoosted.jetSrc = cms.InputTag('boostedTauSeeds')
-    process.combinatoricRecoTausBoosted.modifiers.remove(process.combinatoricRecoTausBoosted.modifiers[3])
```

Comment:
removal of this line breaks compatibility for boostedTau reco for all eras.

I do not consider this change an obvious bugfix addressing something that was completely broken.

So, I propose to either reinstate this "modifiers.remove" or to apply it only to the new eras for which this PR is expected to be used in this 94X release (run2_miniAOD_80XLegacy, run2_miniAOD_94XFall17)